src/mrinufft/trajectories/trajectory3D.py

```diff
@@ -728,6 +729,9 @@ def initialize_3D_annular_shells(
     """Initialize 3D trajectories with annular shells.
 
     An exclusive trajectory inspired from the work proposed in [HM11]_.
+    It is similar to other trajectories based on concentric rings but
+    rings are split and recombined by pairs to better balance
+    the shot lengths.
```

Comment:
I dont fully understand this... split and recombined by pairs? 

Also, can I say that earlier implementation was wrong as this function didn't exist, or did this function make this implementation better?

Reply:
Updated, let me know if it's more clear :)